themes/aframe/layout/blog.ejs

```diff
@@ -8,27 +8,56 @@
 <div class="content content--blog copy post">
   <div class="copy__wrap">
     <% if (page.blog_index) { %>
-
       <h1 class="page__title"><%- config.blog_title %></h1>
 
       <%- page.content %>
 
       <% site.posts.sort('date', -1).each(function (post) { %>
-        <article class="post">
-          <h2 class="post__heading"><%- post.title %></h2>
-          <h4 class="post__timestamp"><%- post.date.format('MMM D[,] YYYY') %></h4>
-          <div class="post__excerpt"><%- post.excerpt %></div>
+        <article class="post post-listing">
+          <a href="<%- page_url(post.path) %>">
+            <h2 class="post__heading"><%- post.title %></h2>
+          </a>
+          <%- partial('partials/blog/metadata', {post: post}) %>
+
+          <% if (post.awoa) { %>
+            <p>What's up with A-Frame for the week of <%- blog_date_subtract_week(post.date) %> to <%- blog_date(post.date) %>.</p>
```

Comment:
change `<%-` to `<%=` (see `Tags` section of [EJS Docs](http://ejs.co/#docs) - I actually learned a few things here)